Header Header
sensor_msgs/Imu imu
sensor_msgs/MagneticField mag_field
string imu_raw_data
